Add example for proxy.istio.io/config annotation (#7348)
istio/istio.io: content/en/docs/tasks/observability/distributed-tracing/configurability/index.md

Then, you can append any configuration options to the `tracing.yaml` file.

> ## Add `istio.proxy.io/config` annotation to your Pod metadata specification

Reviewer: (here and elsewhere): shouldn't this be `proxy.istio.io/config` ?

Author: addressed

> There are occasions where you may wish to override the mesh-wide setting for a Pod-specific

Reviewer 2: ```suggestion
There are occasions where you may wish to override the mesh-wide configuration for a Pod-specific
```

setting. By adding the `proxy.istio.io/config` annotation to your Pod metadata
specification you can override any mesh-wide tracing settings.

For instance, to modify the `sleep` deployment shipped with Istio you would add the following
to `samples/sleep/sleep.yaml`:

{{< text yaml >}}
apiVersion: apps/v1
kind: Deployment
metadata:
  name: sleep
spec:
  ...
  template:
    metadata:
      ...
      proxy.istio.io/config: |
        tracing:
          ...                            # custom tracing definition
    spec:
      ...
{{< /text >}}

> Then, you can append any configuration specified as mentioned below to change tracing settings

Reviewer 2: ```suggestion
You can then append any of the settings specified below, to change the tracing configuration
```
